language/move-lang/src/expansion/translate.rs

```diff
@@ -210,36 +196,43 @@ fn definition(
     }
 }
 
-fn unbound_address_error(suggest_declaration: bool, loc: Loc, n: &Name) -> Diagnostic {
-    let mut msg = format!("Unbound address '{}'", n);
+fn address_without_value_error(suggest_declaration: bool, loc: Loc, n: &Name) -> Diagnostic {
```

Comment:
Ah this will work out great! as long as UnboundAddress is a non-blocking error. and will clean up a lot of other code. I think you can remove the errors from expnasion::ast::into_bytes or whatever it is called